docs/src/pages/components/date-range-picker/BasicDateRangePicker.js

```diff
@@ -20,7 +20,7 @@ export default function BasicDateRangePicker() {
         renderInput={(startProps, endProps) => (
           <React.Fragment>
             <TextField {...startProps} variant="standard" />
-            <DateRangeDelimiter> to </DateRangeDelimiter>
+            <Box sx={{ mx: 2 }}>to</Box>
```

Comment:
```suggestion
            <Box sx={{ mx: 2 }}> to </Box>
```
Might be responsible for the visual regression.

Reply:
I suggest we should do this change in the TS file rather than doing it directly in the JS file.

Reply:
The change needs to happen in both files anyway. The JS file just was the first one in the list.

Reply:
Correct. I've created a new commit for your requested change covering both files.